### PR TITLE
[WIP] Dance page

### PR DESCRIPTION
**Pull request by nkiruka**
WIP - some documents will be deleted (previous approach was to build landing page using react).

-Changed title of first block (Try a second hour fo code to try another hour) to ensure consistency with page mockup

### CODE REVIEW

**Comment by Erin007** [comment]
What's pegasus/sites.v3/code.org/public/shared-templates.haml?

File: pegasus/sites.v3/code.org/public/dance-landing.haml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+%h1= I18n.t(:dance_main_heading)
+
+.dance-subheading
+  %h4= I18n.t(:dance_subheading)
+
+%link{href: "/css/starwars.css", rel: "stylesheet"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/dance-landing.haml:6]: My extraction suggestion is to make a more general haml partial that has generic css, something like full-width-hoc-promo (or whatever you want to call it) that you parameterize and use for minecraft, starwars and dance.  Here you're still reusing starwars.css.  Is that your question?

File: pegasus/sites.v3/code.org/public/dance-landing.haml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      %h2.dance-h2= I18n.t(:dance_party_title)
+      .dance-tutorial-info
+        %p.dance-tutorial-description= I18n.t(:dance_party_description)
+        %p.dance-tutorial-specs= I18n.t(:dance_party_specs)
+      %a{:href => '/api/hour/begin/starwars', :target=>'_self'}
+        %button.dance-try-button= I18n.t(:try_now)
+
+  .col-25.dance-tutorial.dance-blocks-tutorial
+    .tutorial-box
+      %h2.dance-h2 Try Another Hour
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/dance-landing.haml:26]: i18n?

**Comment by Erin007** [comment]
After removing the un-used code, I would submit for review the dance-landing specific css you have and the skeleton outline of the haml as a first PR. Then, you can build out the generic component for the first section (if you still think you'll have time).

File: pegasus/sites.v3/code.org/public/dance-landing.haml (hunk 2 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance-landing.haml:26]: The title of the block used in the mockup increased the size of the block.  This is a potential fix for issue.  Awaiting confirmation from product team.

File: pegasus/sites.v3/code.org/public/dance-landing.haml (hunk 1 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance-landing.haml:6]: Yes, this was part of my question.  Initially, I imported styling from star wars.  Then I added a style sheet for dance-landing.  I created the generic shared-css file to address duplicate styles for the landing pages.

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/dance-landing.haml:6]: I don't see that "generic shared-css" file in this PR - should I?

**Comment by Erin007** [comment]
Maybe my comment above was confusing.  Your page will still have dance-landing specific css, but that dance-landing specific css should NOT include styling for the components (such as the first box in the spec) you are re-using from elsewhere. The dance-landing specific css will be styling for elements that are unique to your page and its particular layout.

**Pull request closed without merging**